core/core-dotspacemacs.el

```diff
@@ -455,17 +455,19 @@ Called with `C-u C-u' skips `dotspacemacs/user-config' _and_ preleminary tests."
                 (load-file buffer-file-name)
                 (dotspacemacs|call-func dotspacemacs/init
                                         "Calling dotfile init...")
-                (dotspacemacs|call-func dotspacemacs/user-init
-                                        "Calling dotfile user init...")
+                (let (spacemacs--keybinding-collision-check nil)
```

Comment:
@bmag  Welp it works fine :confused:  
But you are right `(let (spacemacs--k-c-c) ...)` will be shorter. Thanks :wink: